apps/editor/src/app/atoms/modelTitle.tsx

```diff
@@ -68,18 +68,13 @@ export function ModelTitle({
     <FlexboxGrid align="middle" style={{paddingRight: '5px', paddingBottom: '20px'}}>
       {/* title */}
       <FlexboxGrid.Item colspan={12}>
-        <Row>
-          <Col xs={2} style={{paddingTop: '3px'}}>
-            <Link to={closePath}>
-              <h1>
-                <MdChevronLeft />
-              </h1>
-            </Link>
-          </Col>
-          <Col xs={16}>
-            <h2>{titleView()}</h2>
-          </Col>
-        </Row>
+        <FlexboxGrid align="middle">
```

Comment:
Fixes proper alignment, a11y & html validity (as a chevron was an h1 instead of the title)